tools/packaging/packaging.psd1

```diff
@@ -2,7 +2,7 @@
 GUID="41857994-4283-4757-a932-0b0edb104913"
 Author="Microsoft Corporation"
 CompanyName="Microsoft Corporation"
-Copyright="© Microsoft Corporation. All rights reserved."
+Copyright="(c) Microsoft Corporation. All rights reserved."
```

Comment:
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."